benchmarking/commons/benchmark_definitions/yahpo.py

```diff
@@ -20,15 +20,40 @@
 # which is not supported yet.
 
 # Note: We do not include the `fcnet` scenario. FCNet is a tabulated benchmark
-# evaluated on a fine grid, so surrogate modelling is not needed. It is well
-# served by our own implementation.
+# evaluated completely on a fine grid, so does not profit from surrogate
+# modelling. Our own implementation works fine and provides for much faster
+# simulations.
 
 
-# TODO:
-# - Check that elapsed_time_attr is always cumulative
-# - Check that metric is validation, what we want
-# - Run 2 baseline methods with all these benchmarks, in order to figure out
-#   the most interesting ones
+# -----
+# nb301
+# -----
+
+
+def yahpo_nb301_benchmark(dataset_name):
```

Comment:
NIT (naming): This goes to all the benchmark definition functions in the file. I would suggest renaming them as `yahpo_nb301_benchmark` -> `get_yahpo_nb301_benchmark` to better represent the function of each method.

Reply:
We typically avoid these no-info terms like "get". In the end, this is just meta-data for a benchmark.

Reply:
Makes sense, if thats the general strategy then LGTM.